source/legacy.txt

```diff
@@ -17,7 +17,10 @@ Overview
 --------
 
 In this section, you can identify the changes you need to make to migrate from
-the legacy API to the current API.
+the legacy API to the current API. This page also highlights features unique to 
```

Comment:
S: For accuracy + consistency with our other pages, I'd change the first line to "On this page..." instead of "this section". You can then change "This page also highlights..." to "You can also learn about features unique to..."